client/packages/common/src/intl/utils/DateUtils.ts

```diff
@@ -195,6 +197,34 @@ export const useFormatDateTime = () => {
       : '';
   };
 
+  /**
+   * While getDateOrNull is naive to the timezone, the timezone will still change.
```

Comment:
Nit, maybe a short sentence about what it does first, e.g. : 

"Converts a local date to a Date that has the same date in UTC, e.g. 2024-02-14 becomes 2024-02-14T10:00."

(was this the example problem for Cook Islands?)